api/api/specs/jobs.yaml

```diff
@@ -419,6 +419,13 @@ components:
           type: string
         resources:
           $ref: "#/components/schemas/EnsemblingResources"
+        env:
+          type: array
+          items:
+            $ref: "#/components/schemas/EnvVar"
```

Comment:
You should use `common.yaml#/components/schemas/EnvVar` instead, no need to create a local copy of it. 
The reason why it was done before for `Id` schema is because external refs were not resolved properly from within `path` parameter section (i.e.https://github.com/gojek/turing/blob/992d465ceb24a9f7e84a7c3346bed982a58862c6/api/api/specs/jobs.yaml#L19).

Reply:
done